web-client/iron-remote-gui/src/services/wasm-bridge.service.ts

```diff
@@ -75,7 +75,7 @@ export class WasmBridgeService {
     }
 
     /// Callback to set the local clipboard content to data received from the remote.
-    setOnRemoteClipboardChanged(callback: OnRemoteClipboardChanged) {
+    setOnRemoteClipboardChanged(callback: OnRemoteClipboardChanged | null) {
         this.onRemoteClipboardChanged = callback;
     }
```

Comment:
I think the right way to do what you need on DVLS side, is to not call this function at all. Do you think this is problematic?